### ISSUE TITLE
QEMU: SHA256 sum mismatch on VENOM patch

### ISSUE THREAD

**Issue by mtpereira** [Homebrew/homebrew#41740]
Hello,

I'm getting this error when installing QEMU: https://gist.github.com/anonymous/2e067a0e1c586cb8253d

I've tried curling the patchfile and I get the same "Actual:" sum from brew:

```
$ curl 'http://git.qemu.org/?p=qemu.git;a=commitdiff_plain;h=e907746266721f305d67bc0718795fedee2e824c' 2>/dev/null | sha256sum
2aaa9ff9ee492dede64df3fcb59032ae0452bf4790c2d5881e05981fa7452368  -
```

Did anyone else got this? Am I doing something wrong here?

Thank you.

**Comment by bfontaine** [comment]
I can reproduce.

**Comment by DomT4** [comment]
Yup, here too. We use `commitdiff_plain` because it's supposed to stop the patch breaking whenever upstream update their `git` version or similar; not sure why that hasn't prevented the SHA changing here.

**Comment by dunn** [comment]
Fixed by #41754. Forgot to add that to the commit message, my bad!
